config/samples/helm-charts/varnishcluster-with-monitoring/templates/varnish-cluster.yaml

```diff
@@ -11,7 +11,16 @@ spec:
     entrypointFileName: entrypoint.vcl
   backend:
     selector:
-      {{ toYaml .Values.varnish.backendsSelector | nindent 6 }}
-    port: {{ .Values.varnish.backendsPort }}
+      app: backend
+    port: backend
   service:
     port: 80
+  monitoring:
+    prometheusServiceMonitor:
+      enabled: true
+      scrapeInterval: 35s
+      labels:
+        app: varnish
+    grafanaDashboard:
+      enabled: true
+      datasourceName: Prometheus-varnish
```

Comment:
Where does this value come from? I set it to `Prometheus` in the past, but not sure why.

Reply:
It comes from [here](https://github.com/IBM/varnish-operator/blob/main/config/samples/helm-charts/varnishcluster-with-monitoring/templates/grafana-prometheus-datasource.yaml#L13).
You can name it as you want, just make sure those values are the same here and in the datasource definition.

Reply:
*cough* newline *cough*